sqs/src/test/scala/docs/scaladsl/SqsPublishSpec.scala

```diff
@@ -191,6 +197,28 @@ class SqsPublishSpec extends AnyFlatSpec with Matchers with DefaultTestContext w
     }
   }
 
+  it should "publish batch of SendMessageRequests with individual delays and pull empty seq" taggedAs Integration in {
+    new IntegrationFixture {
+      // #batch-send-request
+      val messages = for (i <- 0 until 10)
+        yield
+          SendMessageRequest
+            .builder()
+            .messageBody(s"Message - $i")
+            .delaySeconds(60) // SQS queue should process messages after 60 seconds
```

Comment:
Is there any chance that this could be set to something like 5 seconds and you could check that no messages arrive within 3 seconds (5 minus some lambda) and then check if the message arrives within the following 10 seconds? These numbers mean the test won't take too long to run but should hopefully be pretty stable.

Reply:
Done, it took me a bit of time because I couldn't realize why the request to the SQS queue didn't want to wait at all.

The issue was in the `receiveMessages` method. Wait time has to be specified in the SQS client request.

Reply:
Also, the reason why I didn't want to do something like

```
      receiveMessages(10, messageDelaySeconds - 1) should have size 0 // delay hasn't elapsed
      receiveMessages(10, messageDelaySeconds + 1) should have size 10 // delay has elapsed
```

Is that both of these calls block the thread and the total amount of waiting would be 2 x messageDelaySeconds.
I wanted to make both of these method calls async, but that would require creating another `recieveMessages` function which would return a future, and that handling this in another way.

Decided to not go with that approach cause it seemed more complicated, but can do it if you think there's need to do so.